rust/datafusion/src/execution/context.rs

```diff
@@ -190,8 +191,10 @@ impl ExecutionContext {
             Box::new(TypeCoercionRule::new()),
         ];
         let mut plan = Arc::new(plan.clone());
+        println!("optimizing plan: {:?}", plan);
```

Comment:
I assume that these are used for debugging, so maybe it's better to use logger?

Reply:
I removed for now, since we don't have a logger dependency yet. I would like to add one though. I'll create a JIRA.